### ISSUE TITLE
WIP Won't compile in 32bit

### ISSUE THREAD

**Issue by JulienMaille** [wangwenx190/framelesshelper#109]
This is due to `GetClassLongPtrW` not being defined
```
Error	C2065	'kGetClassLongPtrW': undeclared identifier	FramelessHelper\src\core\utils_win.cpp	930	
Error	C2065	'kGetWindowLongPtrW': undeclared identifier     FramelessHelper\src\core\utils_win.cpp 942
```
I suggest something like this
```cpp
#ifndef _WIN64
#define kGetClassLongPtrW kGetClassLongW
#define kSetClassLongPtrW kSetClassLongW
#define kGetWindowLongPtrW kGetWindowLongW
#define kSetWindowLongPtrW kSetWindowLongW
#endif
```
or maybe just
```cpp
FRAMELESSHELPER_STRING_CONSTANT2(GetClassLongPtrW, "GetClassLongPtrW")
FRAMELESSHELPER_STRING_CONSTANT2(SetClassLongPtrW, "SetClassLongPtrW")
FRAMELESSHELPER_STRING_CONSTANT2(GetWindowLongPtrW, "GetWindowLongPtrW")
FRAMELESSHELPER_STRING_CONSTANT2(SetWindowLongPtrW, "SetWindowLongPtrW")
```

**Comment by wangwenx190** [comment]
Yes the `Ptr` suffixed version doesn't exist on 32bit platforms, I'll fix this issue later tonight. Sorry for the inconvenience!

**Comment by wangwenx190** [comment]
Hi, the code has been updated. Can you check whether this issue has been fixed by commit eb1c6f4a62e903305f81da8acacdb3fb3b62a603 or not? Thanks!

**Comment by wangwenx190** [comment]
Sorry I made a mistake, commit eb1c6f4 didn't fix the issue actually. It's verified commit 3a46649 fixed the issue for real.
